core/metacling/src/TCling.cxx

```diff
@@ -6934,6 +6934,14 @@ void TCling::CodeComplete(const std::string& line, size_t& cursor,
    fInterpreter->codeComplete(line, cursor, completions);
 }
 
+std::pair<std::string, std::string> TCling::TInterpreterValue_SplitTypeValue(TInterpreterValue& value)
+{
+   std::string output = value.ToString();
+   size_t pos = output.find("&") + 2;
```

Comment:
What about `gROOT`? Or `(int*)nullptr`? There will be no '&'.

You could probably cut on the type output's trailing `)`?